src/main/java/net/minecraftforge/items/wrapper/SidedInvWrapper.java

```diff
@@ -233,6 +233,19 @@ public int getSlotLimit(int slot)
     public boolean isItemValid(int slot, @NotNull ItemStack stack)
     {
         int slot1 = getSlot(inv, slot, side);
-        return slot1 == -1 ? false : inv.canPlaceItem(slot1, stack);
+        return slot1 == -1 ? false : inv.canPlaceItem(slot1, stack); // TODO: 1.20 - Change this to the face-aware method
```

Comment:
Couldn't this be considered a bug in Forge, and as such anyone relying on the old behavior would be doing so wrongly? Hence it wouldn't be a breaking change.

Reply:
It's both a breaking bug and a breaking fix, so I think this one will need attention from triage.